jobs/JEVS_NFCENS_WAVE_GRID2OBS_PLOTS

```diff
@@ -42,7 +43,6 @@ export USHevs=${USHevs:-$HOMEevs/ush}
 ####################################
 # Define NET/RUN variables          
 ####################################
```

Comment:
@SamiraArdani-NOAA Could the entire "Define NET/RUN variables" section be moved before "Set EVS directories"? I don't think we want to move "NET" out of the the "Define NET/RUN variables" section. I think this whole block could be moved before the "Set EVS directories" section so that NET can be used in HOMEevs.

CC @PerryShafran-NOAA @ShelleyMelchior-NOAA 

```
####################################
# Define NET/RUN variables          
####################################
export NET=${NET:-evs}
export MODEL=${MODEL:-evs}
export STEP=${STEP:-plots}
export COMPONENT=${COMPONENT:-nfcens}
export MODELNAME=${MODELNAME:-nfcens}
export MODNAM=$(echo $MODELNAME | tr '[a-z]' '[A-Z]')
export OBTYPE=${OBTYPE:-GDAS}
export RUN=${RUN:-wave}
export VERIF_CASE=${VERIF_CASE:-grid2obs}

####################################
# Set EVS directories               
####################################
export HOMEevs=${HOMEevs:-${PACKAGEROOT}/${NET}.${evs_ver}}
export EXECevs=${EXECevs:-$HOMEevs/exec}
export FIXevs=${FIXevs:-$HOMEevs/fix}
export PARMevs=${PARMevs:-$HOMEevs/parm}
export USHevs=${USHevs:-$HOMEevs/ush}
```